Makefile

```diff
@@ -55,12 +55,28 @@ qct:
 	else ct_run -pa apps/*/ebin -pa deps/*/ebin -pa ebin -dir apps/*/test\
         -I apps/*/include -I apps/*/src -logdir /tmp/ct_log -noshell; fi
 
-test: test_deps
+test: test_node_stopped test_deps
 	cd test/ejabberd_tests; make test
 
-test_preset: test_deps
+# $ make qtest SUITE=mam_SUITE
+# $ make qtest SUITE="[mam_SUITE,metrics_api_SUITE]"
+qtest: test_node_stopped check_cfg_backup
```

Comment:
There is no consistent way:

```
make ct SUITE=cassandra
make qct SUITE=amp_resolver_SUITE
```

Also, it's possible to add prefix for each element of the list, but requires extra script to parse erlang term.